packages/make-angular-cli-faster/src/utilities/make-angular-cli-faster.ts

```diff
@@ -42,4 +43,27 @@ export async function makeAngularCliFaster(args: Args) {
       'and how it can speed up your CI by up to 10 times at https://nx.dev/getting-started/nx-and-angular',
     ],
   });
+
+  if (migration.incompatibleWithAngularVersion) {
+    output.warn({
+      title:
+        'The provided version of Nx is not compatible with the currently installed version of Angular.',
```

Comment:
I thought we ran an older migration? 

The "provided version" is the version of `make-angular-cli-faster` that was run right?

But we choose an older version of `@nrwl/workspace` or `@nrwl/angular` to use which they have migrated to?

So the version of Nx they ended up should be compatible right?

This might cause confusion when people think that running `make-angular-cli-faster` gave them a version of Nx which is incompatible.

Reply:
This is when the user provided a version when running the script with `--version`. So, they explicitly decided to run it with a specific version. Before, we were handling that and proposing a compatible version to use, but as per @vsavkin request, that was removed 🤷🏻 

Let's discuss this offline. If you ask me, we should at least fail and not let the migration continue even if it was the user choosing that version.

Reply:
Mention the exact version that they passed and the version of Angular that they have.